etl/steps/data/garden/homicide/2023-01-04/unodc.meta.yml

```diff
@@ -48,15 +48,220 @@ dataset:
 tables:
   unodc:
     variables:
-      counts:
-        title: Homicides
+      firearms_homicide_count:
+        title: Total homicides by firearm
         short_unit: ''
```

Comment:
You can skip `short_unit` and just not declare it. I believe this gets rendered to `''` in Grapher.